src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown

```diff
@@ -0,0 +1,545 @@
+---
+layout: updates/post
+title: "Creating a Web-enabled IoT device with Intel&reg; Edison"
+description: "Internet of Things is really on everyone's lips these
+days, but what if you could create an IoT device which you accessed
+from your web browser? In this article we look at how you can do
+exactly that using Physical Web, Web Bluetooth as well as Node.js,
+running on an Intel&reg; Edison Arduino breakout board."
+published_on: 2016-03-20
+updated_on: 2016-03-20
+authors:
+  - kenchris
+tags:
+  - news
+  - Bluetooth
+  - IoT
+  - WebBluetooth
+  - PhysicalWeb
+  - Edison
+featured_image: /web/updates/images/2016/03/web-enabled-internet-of-things/board.jpg
+---
+
+<p class="intro">
+Internet of Things is really on everyone's lips these days, and
+it makes a tinkerer and a programmer like me very excited. Nothing
+is much cooler than bringing your own inventions to life, and
+being able to talk to them!
+</p>
+
+![Client application](/web/updates/images/2016/03/web-enabled-internet-of-things/tablet-ux.png)
+
+# Web and IoT, a match to be
+
+There are still a lot of hurdles to be overcome before Internet
+of Things will be a huge success. Personally, I am worrying about
+companies requiring people to install their apps for each device
+they produce, cluttering my phone with a multitude of apps that
+I *rarely* use. Something that I would like to avoid.
+
+For this reason, I have been very excited about the
+[Physical Web](https://google.github.io/physical-web/)
+project, which allows devices to broadcast a URL to an online
+website, in a *non-intrusive way*. In combinations with emerging
+web technologies such as
+[Web Bluetooth](https://webbluetoothcg.github.io/web-bluetooth/),
+[Web USB](https://wicg.github.io/webusb/) and
+[Web NFC](https://w3c.github.io/web-nfc/), the site can either
+connect directly to the device or at least explain the proper
+way of doing so.
+
+Web Bluetooth would be a good way of doing so, but in some cases
+you might require a physical connection (say via USB) for security
+reasons, for instance in an office setting.
```

Comment:
Suggestion: instead of `in an office setting`, what about using `public utilities` as an example.